vshard/storage/init.lua

```diff
@@ -1196,20 +1207,28 @@ local function recovery_f()
             goto sleep
         end
 
+        service:set_activity('recovering sending buckets')
         ok, total, recovered = pcall(recovery_step_by_type,
                                      consts.BUCKET.SENDING)
         if not ok then
             is_all_recovered = false
-            log.error('Error during sending buckets recovery: %s', total)
+            local err_msg = string.format(
+                'Error during sending buckets recovery: %s', total)
+            service:set_status_error(err_msg)
+            log.error(err_msg)
         elseif total ~= recovered then
             is_all_recovered = false
         end
 
+        service:set_activity('recovering receiving buckets')
```

Comment:
Extra picky nit: I would reword it to `recovery recv` or `recovery receiving`. It is obvious that these are "buckets" anyway.

Reply:
Fixed